### PR TITLE
Added Onboarding Sequence and local database

### PR DESCRIPTION
**Pull request by usersblock**
🚀 Pull Request
Advances Issue #17 #13 
Form now has an on-boarding sequence that is triggered depending on if a debug variable is true or false. In addition, the on-boarding sign-up sequence sends an object to a local database.

Describe the solution you implemented
#17 
The on-boarding sequence is the login and sign-up options that appears first when the user first enters. This screen is skipped and the user reaches the app screen if the on-boarding var is set to FALSE. In the (finalSequence(array,array,array)) function of the component, the function accepts parameters that the user inputs and creates a JSON object to be sent to a local data base in expo.

#13 
Header text now aligns on the Y-Axis and the Skip button is centered.

How has this been tested?
Visually (Second screen is first when on-boarding is false and last when is true)

Screen shots
image
image
image

### Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: fytProject/components/FormTrain.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
+import {ONBOARDING} from '../config/settings';
+import { FileSystem } from 'expo';
 import { Animated, Dimensions, StyleSheet } from 'react-native';
 import { Button, Card, Text, View, Form, Item, Input } from 'native-base';
+const onboardingvar=ONBOARDING;
```
Comment (DanielMConnolly) [inline fytProject/components/FormTrain.js]: Is there a reason you are assigning this here as opposed to just using the onboarding variable?

File: fytProject/components/FormTrain.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
+import {ONBOARDING} from '../config/settings';
+import { FileSystem } from 'expo';
 import { Animated, Dimensions, StyleSheet } from 'react-native';
 import { Button, Card, Text, View, Form, Item, Input } from 'native-base';
+const onboardingvar=ONBOARDING;
+var startstep="LOGINSIGNUP";
```
Comment (DanielMConnolly) [inline fytProject/components/FormTrain.js]: The startstep strings should be put in the settings file as well.

File: fytProject/components/FormTrain.js [hunk 3]
```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
+import {ONBOARDING} from '../config/settings';
+import { FileSystem } from 'expo';
 import { Animated, Dimensions, StyleSheet } from 'react-native';
 import { Button, Card, Text, View, Form, Item, Input } from 'native-base';
+const onboardingvar=ONBOARDING;
+var startstep="LOGINSIGNUP";
+if (onboardingvar==false){
```
Comment (DanielMConnolly) [inline fytProject/components/FormTrain.js]: Like here for instance we should just have ```if(ONBOARDING){}```

**Comment by RychardHunt** [comment]
closes #17 
closes #13

**Pull request merged**